config/sync/core.entity_form_display.node.author.default.yml

```diff
@@ -0,0 +1,52 @@
+uuid: 0f1712d5-b377-47a2-9e20-18376f94bdee
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.author.field_a_biography
+    - field.field.node.author.field_a_image
+    - node.type.author
+  module:
+    - media_library
+id: node.author.default
+targetEntityType: node
+bundle: author
+mode: default
+content:
+  field_a_biography:
+    weight: 2
+    settings:
+      rows: 5
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textarea
+    region: content
+  field_a_image:
+    type: media_library_widget
+    weight: 3
+    settings:
+      media_types: {  }
+    third_party_settings: {  }
+    region: content
+  status:
+    type: boolean_checkbox
+    settings:
+      display_label: true
+    weight: 4
+    region: content
+    third_party_settings: {  }
+  title:
+    type: string_textfield
+    weight: 0
+    region: content
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+hidden:
+  created: true
+  moderation_state: true
+  path: true
+  promote: true
+  sticky: true
+  uid: true
```

Comment:
Please ensure that these fields are displayed and also add the author field to accepted content types in content moderation.